documentation/source/coroutines.rst

```diff
@@ -151,3 +151,35 @@ they'd naturally end.
         if not isclose(edge_time_ns, start_time_ns + 4.0):
             raise TestFailure("Expected a period of 4 ns")
 
+
+Async functions
+---------------
+
+Python 3.5 introduces ``async`` functions, which provide an alternative syntax.
+For example:
+
+.. code-block:: python
+
+    @cocotb.coroutine
+    async def wait_10ns():
+        cocotb.log.info("About to wait for 10ns")
```

Comment:
nit: "10 ns" (for consistency with the message below. You might want to change that in the same example above as well)

Reply:
Copied directly from the example above.

Reply:
Fixed anyway